tests/lean/univ_of_instances.lean

```diff
@@ -0,0 +1,14 @@
+section
+universes v u
+
```

Comment:
Tests that don't have any output and are only required to compile successfully should go into the `tests/lean/run` subdirectory.

Reply:
How do I run the test then?

Is there any documentation for the test system? It's impossible to discover anything under `tests/lean` since there are 1000+ files there.

Reply:
I don't know of any documentation.  The canonical way to run the tests is to run `make test` or `ctest` in the build directory.  The nice thing about the files in the `tests/lean/run` directory is that they only need to run successfully, there is no complicated setup.

Reply:
Ah, thanks. I didn't even know about `make test`. The cmake build system has some annoying caching which made this much more difficult to test locally than it needed to be, but I think I succeeded in moving the test.